tests/ecmp/test_ecmp_sai_value.py

```diff
@@ -152,15 +152,23 @@ def check_ecmp_offset_value(duthost, asic_name, topo_type, hwsku):
         offset_count = offset_list.count('0')
         if asic_name == "td3":
             # For TD3, RTAG7_PORT_BASED_HASH.ipipe0[1]: <OFFSET_ECMP=2,>
-            pytest_assert(offset_count == 391, "the count of 0 OFFSET_ECMP is not correct.")
+            pytest_assert(offset_count == 391, "the count of 0 OFFSET_ECMP is not correct. \
+                          Expected {}, but got {}.".format(391, offset_count))
+        elif hwsku in ["Arista-7050-QX-32S", "Arista-7050QX32S-Q32"]:
```

Comment:
Could we keep consistence here to use either asic_name for both cases? The 7050qx asic should be td2. If we don't have those information, could we use hwsku for both cases?

Reply:
@StormLiangMS Done.